treeherder/intermittents_commenter/commenter.py

```diff
@@ -275,11 +276,19 @@ def get_bug_stats(self, startday, endday):
                     "mozilla-inbound": 3
                 },
                 "per_platform": {
-                    "osx-10-10": 4,
-                    "b2g-emu-ics": 1
+                    "windows10-64": 52,
+                    "osx-10-10": 1,
```

Comment:
we don't run osx-10-10 anymore, more possible cleanup, it is replaced by osx-10-14 (and soon to be osx-10-15, but that could take a few months to get on all branches)

Reply:
Sure, I can update those code comments.